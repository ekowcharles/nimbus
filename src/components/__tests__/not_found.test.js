import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFound from '../not_found'

describe('NotFound', () => {
  it('renders page with correct message when route is not found', () => {
    const wrapper = shallow(<NotFound />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})