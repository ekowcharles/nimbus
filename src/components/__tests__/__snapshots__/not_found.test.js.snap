// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders page with correct message when route is not found 1`] = `
<div
  className="area-not-found"
>
  <p>
    <b>
      Sorry!
    </b>
     The page you are looking for does not exist.
  </p>
</div>
`;
